ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -174,19 +174,8 @@ tests:
         OPENSHIFT_VERSION=4.14
         OPENSHIFT_INSTALL_RELEASE_IMAGE=""
     workflow: assisted-baremetal
-- as: e2e-metal-assisted-4-12-periodic
```

Comment:
4.12 is still in extended support, we may want to keep it? https://access.redhat.com/support/policy/updates/openshift

Reply:
I thought just leaving the presubmit, but yes you are right

Reply:
Fixed, thank you